packages/components/src/form-token-field/stories/index.story.tsx

```diff
@@ -99,6 +100,7 @@ export const Async: StoryFn< typeof FormTokenField > = ( {
 Async.args = {
 	label: 'Type a continent',
 	suggestions: continents,
+	__nextHasNoMarginBottom: true,
```

Comment:
Confirmed that all stories inherit args from either of the 2 definitions that define `__nextHasNoMarginBottom: true`.